xml/System.Reflection/MethodSemanticsAttributes.xml

```diff
@@ -167,7 +167,7 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary><para>Used to modify the value of the property.</para><para>CLS-compliant setters are named with the <see langword="set_" /> prefix.</para></summary>
```

Comment:
@rpetrusha Thanks for letting me know that enum values cannot have a remark. What is the purpose of the `<para>` tags you added?